source/dlangbot/github_api.d

```diff
@@ -140,20 +140,20 @@ struct PullRequest
     @byName GHState state;
     uint number;
     string title;
-    Nullable!bool mergeable;
-    @byName @name("mergeable_state") Nullable!MergeableState mergeableState;
+    @optional Nullable!bool mergeable;
+    @optional @byName @name("mergeable_state") Nullable!MergeableState mergeableState;
```

Comment:
The new vibe-d payload doesn't contain `mergable` and the Vibe.d side of things currently [looks like this](https://github.com/rejectedsoftware/vibe.d/blob/4d952f7abdf72098a4e1919cfd18b8a6ec9213f7/data/vibe/data/serialization.d#L763):

```d
foreach (i, mname; SerializableFields!(T, Policy))
	static if (!hasPolicyAttribute!(OptionalAttribute, Policy, __traits(getMember, T, mname)))
		enforce(set[i], "Missing non-optional field '"~mname~"' of type '"~T.stringof~"' ("~Policy.stringof~").");
```